docs/architecture/providers/parameterized.md

```diff
@@ -1,14 +1,46 @@
 (parameterized-providers)=
 # Parameterized providers
 
-*Parameterized providers* are a feature of Pulumi that allows a caller to change
-a provider's behaviour at runtime in response to a
-[](pulumirpc.ResourceProvider.Parameterize) call. Where a
-[](pulumirpc.ResourceProvider.Configure) call allows a caller to influence
-provider behaviour at a high level (e.g. by specifying the region in which an
-AWS provider should operate), a [](pulumirpc.ResourceProvider.Parameterize) call
-may change the set of resources and functions that a provider offers (that is,
-its schema). A couple of examples of where this is useful are:
+*Parameterized providers* are a provider feature where a user can generate and
+use a custom SDK based on some custom input.
```

Comment:
This isn't strictly true I don't think -- the set of use cases is a bit wider than this (though of course this is a common user-visible artifact of it) and I do think the original wording is closer to the truth here. That's not to say we shouldn't talk about custom SDKs, etc., but that's more around `pulumi package add`/the consequences of parameterization, if that makes sense, rather than parameterization itself.